modules/raft/src/main/java/org/apache/ignite/internal/raft/Loza.java

```diff
@@ -89,8 +89,9 @@ public Loza(ClusterService clusterNetSvc, Path dataPath) {
 
         String locNodeName = clusterNetSvc.topologyService().localMember().name();
 
-        if (nodes.stream().map(ClusterNode::name).collect(Collectors.toSet()).contains(locNodeName))
+        if (nodes.stream().map(ClusterNode::name).collect(Collectors.toSet()).contains(locNodeName)) {
```

Comment:
According to our code style we shouldn't use {} for one-liners. It's also strange that inspections didn't cache this - could you please check this?

Reply:
Why you need to copy collection just to check for contains?
```suggestion
        if (nodes.stream().anyMatch(n -> locNodeName.equals(n.name()))) {
```